osquery/sql/tests/virtual_table.cpp

```diff
@@ -1219,24 +1219,11 @@ TEST_F(VirtualTableTests, test_table_exceptions) {
   attachTableInternal(
       "exceptional", exceptional->columnDefinition(false), dbc, false);
 
-  auto backup_flag = FLAGS_table_exceptions;
```

Comment:
While I think we should never attempt to interrupt sqlite's callbacks with exceptions, I would still like to know why it was failing on AArch64; I'll spawn an instance and try to debug this